src/app_ui/sign.rs

```diff
@@ -64,15 +75,41 @@ pub fn ui_display_tx(tx: &Tx) -> Result<bool, AppSW> {
     ];
 
     // Create transaction review
-    let my_review = MultiFieldReview::new(
-        &my_fields,
-        &["Review ", "Transaction"],
-        Some(&EYE),
-        "Approve",
-        Some(&VALIDATE_14),
-        "Reject",
-        Some(&CROSSMARK),
-    );
+    #[cfg(not(any(target_os = "stax", target_os = "flex")))]
+    {
+        let my_review = MultiFieldReview::new(
+            &my_fields,
+            &["Review ", "Transaction"],
+            Some(&EYE),
+            "Approve",
+            Some(&VALIDATE_14),
+            "Reject",
+            Some(&CROSSMARK),
+        );
```

Comment:
Please use the same wording as for C BP (and **no** subtext)

```
    nbgl_useCaseReview(TYPE_TRANSACTION,
                       &pairList,
                       &C_app_boilerplate_64px,
                       "Review transaction\nto send BOL",
                       NULL,
                       "Sign transaction\nto send BOL",
                       review_choice);
```

Reply:
Done.